app/controllers/account/projects_controller.rb

```diff
@@ -6,45 +6,51 @@ def index
 
   def show
     @workspace = parent
-    @project = @workspace.projects.find(params[:id])
+    @project = resource
     @tasks = @project.tasks.row_order_asc
   end
 
   def new
     @workspace = parent
-    @project = current_user.projects.build
+    @project = collection.build
   end
 
   def create
-    if current_user.projects.create(project_params)
-      redirect_to account_workspace_projects_path(parent)
+    @workspace = parent
+    @project = collection.build(project_params)
+    if  @project.save
+      redirect_to account_workspace_projects_path(parent), notice: "Project was created!"
     else
       render :new
     end
   end
 
   def edit
     @workspace = parent
-    @project = @workspace.projects.find(params[:id])
+    @project = resource
   end
 
   def update
+    @workspace = parent
     @project = resource
-
     if @project.update(project_params)
-      redirect_to account_workspace_projects_path(parent)
+      redirect_to account_workspace_projects_path(parent), notice: "Project was edited!"
```

Comment:
updated would sound more truly

Reply:
"Project was successfully updated"

Do similar to other places of App. Be consistent